stackrox/stackrox
# Add various minor Workload CVE fixes

## Description

A couple of minor fixes, one commit for each change:

1. Cleanup of GQL queries to be consistent with our engineering document
2. Update Severity and Fixability icons to use the latest from UX (Wrench, Minus instead of Check, X)
3. Fix layout issue with the filter bar and issue with autocomplete for CVE category
4. Fix incorrect breadcrumb link on CVE single page

## Checklist

If any of these don't apply, please comment below.

## Testing Performed

2) View pages with severity and fixability icons:
![image](https://github.com/stackrox/stackrox/assets/1292638/845c8e2b-d6aa-4f46-80c5-cc0aaaa1a891)

3) Ensure entity dropdown does not extend to take up the entire space, ensure CVE values autocomplete:
![image](https://github.com/stackrox/stackrox/assets/1292638/21b20fe3-352c-4f5b-a1fd-cb3d21ebb18b)
![image](https://github.com/stackrox/stackrox/assets/1292638/568dcef2-41b4-4c28-b441-de34dc91a2ee)

4) Ensure CVE breadcrumb returns you to the overview page on the **CVE** tab
![image](https://github.com/stackrox/stackrox/assets/1292638/e9ab1731-9315-4bb7-9723-676dbb4294ca)
![image](https://github.com/stackrox/stackrox/assets/1292638/fb9eff4f-1e3f-4c39-a940-d51c3850468e)
